main.go

```diff
@@ -89,6 +89,7 @@ func configureVariables() {
 	viper.SetDefault("STRICT", true)
 	viper.SetDefault("ENVIRONMENT_TYPE", "CDK")
 	viper.SetDefault("FQDN", "obdemo-bank.idhub.cc")
+	viper.SetDefault("RCS_UI_FQDN", "rcs-ui.dev.forgerock.financial")
```

Comment:
Hard coded to dev? Should this be an env var and be set in the deployment??

Reply:
Yes, In that case and at the moment is set only in one place because we have only one am instance for all environments. this is the value of redirect URL for the remote consent.